src/Pos/Binary/Txp.hs

```diff
@@ -21,6 +21,10 @@ instance Bi T.TxOut where
     put (T.TxOut addr coin) = put addr >> put coin
     get = label "TxOut" $ T.TxOut <$> get <*> get
 
+instance Bi T.TxOutAux where
+    put (T.TxOutAux out distr) = put out >> put distr
```

Comment:
I have a suspicion that `<>` and `>>` might actually generate equal Core (as in “GHC Core”)